.circleci/config.yml

```diff
@@ -17,6 +17,31 @@ jobs:
       xcode: 11.5.0
     steps:
       - checkout
+      - run:
```

Comment:
I put this in (and removed the Cartfile target) per a discussion with Pete about the options for integrating SwiftCharts. Basically we could use the already-existing SwiftCharts submodule in the workspace, or use Carthage with caching. Since we really only have the Carthage for the purpose of the CI builds, it was recommended that I remove the Cartfile target from the project and add those build steps to the CI build.

Reply:
So how will DIY users build LoopKit after this?

Reply:
Main DIY builds via Carthage through Loop, and for the DIY workspace build, we also include SwiftCharts project directly.

Reply:
Is it okay that DIY Xcode builds from the `LoopKit` Xcode project won't work?

Reply:
I have a similar question - based on how I _think_ Carthage works, wouldn't it be insufficient to have the Loop Carthage be pulling in SwiftCharts?

Reply:
Why not remove SwiftCharts from the list of submodules? We're not changing it.  So let's just pull it in via Carthage, either in DIY or Tidepool builds.  Would that be possible, @ps2 ?

Reply:
I think the concern is speed - Carthage is slower than the submodules, no?

Reply:
Carthage (bootstrap) only needs to run once

Reply:
I have several times modified SwiftCharts; it's a small project, and they've taken a number of DIY's changes.  It's open source, and building it as a first class member of our project is easy enough. And yes, even with caching, it does add significant time on the first build.

Reply:
> Is it okay that DIY Xcode builds from the LoopKit Xcode project won't work?
Without running `carthage bootstrap` first? Yes. That's how several of the DIY dependencies work right now.  The main thing is that the non-workspace build of Loop needs to be able to build everything as a zip, and this setup allows that, and also allows us to have speed and easy access to the SwiftCharts source.

Reply:
> I have a similar question - based on how I think Carthage works, wouldn't it be insufficient to have the Loop Carthage be pulling in SwiftCharts?

Loop Cartfile references LoopKit, which makes Carthage fetch the Cartfile for LoopKit and resolve those dependencies too. So the Loop project, in xcode, when it runs the Cartfile target will end up building SwiftCharts, even if Loop's Cartfile didn't reference it.

Reply:
I guess I'm still not convinced this is the right approach.  So as not to block @novalegra I will approve.